resources/js/components/forms/Widget.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+    <div class="h-full">
+        <div v-if="initializing" class="loading">
+            <loading-graphic />
+        </div>
+
+        <data-list
+            v-if="!initializing && items.length"
+            :rows="items"
+            :columns="cols"
+            :sort="false"
+            :sort-column="sortColumn"
+            :sort-direction="sortDirection"
+            class="flex h-full flex-col justify-between"
+        >
+            <div>
+                <data-list-table :loading="loading">
+                    <template #cell-datestamp="{ row: submission }">
+                        <div class="flex items-center">
+                            <a :href="submission.url">{{ formatDate(submission.datestamp) }}</a>
```

Comment:
I ended up converting the form submissions widget into a Vue component, so I could do the date formatting here. Most of the code is stolen from the collection widget, which works in a similar way.

It does slightly change the look of the widget, but I figured that would be fine since we're about to redesign the Control Panel anyways.